resize-observer/devicepixel2.html

```diff
@@ -0,0 +1,157 @@
+<!doctype html>
+<link rel="match" href="devicepixel2-ref.html">
+<meta name="assert" content="Device pixel content box sizes and pixel snapping are correct in Resize Observer callback">
+<style>
+  html, body {
+    margin: 0;
+    height: 100%;
+  }
+  #back {
+    width: 100%;
+    height: 100%;
+  }
+  #front {
+    position: absolute;
+    left: 0;
+    top: 0;
+  }
+  .outer {
+    width: 99.5px;
+    display: flex;
+  }
+  .outer>* {
+    flex: 1 1 auto;
```

Comment:
Let's just condense this to `flex: 1;` -- that's essentially equivalent (and simpler). The trailing "1 auto" is irrelevant and unused in practice, since the matched "inner" elements don't have any contents and so their "auto" size is 0 anyway.

(See https://drafts.csswg.org/css-flexbox-1/#flex-common which has the spec's suggested abbreviated usages for the `flex` shorthand, with "flex: <number>" being the preferred way to divide-the-free-space-equally.)

Reply:
done